lib/cloud_controller/diego/lifecycles/buildpack_lifecycle.rb

```diff
@@ -57,6 +57,13 @@ def app_stack
       @package.app.buildpack_lifecycle_data.try(:stack)
     end
 
+    def buildpack_stack
+      stacks = Buildpack.where(name: buildpacks_to_use).select(:stack).map(&:stack).uniq
+      if stacks.length == 1
```

Comment:
What should this return if `stacks.length` is > 1?

Reply:
nil